lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -561,10 +566,27 @@ def estimate_diffusion_constant_simple(
     if max_lag < 2:
         raise ValueError("You need at least two lags to estimate a diffusion constant")
 
+    if any(np.diff(frame_idx) > 1):
+        if method == "gls":
+            raise RuntimeError(
+                "Your tracks cannot have missing frames when using the GLS estimator. Refine your "
+                "tracks using `lk.refine_tracks()`. Please refer to `help(lk.refine_tracks)` for "
```

Comment:
```suggestion
                "tracks using `lk.refine_tracks_centroid()`. Please refer to `help(lk.refine_tracks_centroid)` for "
```

Reply:
good cach

Reply:
That reminds me, I was doubting about this a bit. We could also refer to the docs instead. `reifne_tracks_centroid()` chooses a very particular method for refinement. One may want Gaussian.